src/map/lua/luautils.cpp

```diff
@@ -5581,6 +5582,51 @@ namespace luautils
         customMenuContext.erase(PChar->id);
     }
 
+    int SendItemToDeliveryBox(std::string const& playerName, uint16 itemId, uint32 quantity, std::string senderText)
```

Comment:
Should be a uint return, given the results, also, since this lands in lua, the ret results should most likely be enum'd

Reply:
> Should be a uint return, given the results, also, since this lands in lua, the ret results should most likely be enum'd

For the enum'd, do you mean like an enum class in luautils and in a lua enum file under scripts/enum? Also do I need an enum lua object then that is passed between c++ and lua? or can I just use like uint8?

Reply:
> For the enum'd, do you mean like an enum class in luautils and in a lua enum file under scripts/enum?

Yes. You can still return a uint8 with `static_cast<uint8>(your::enum::val)`, in lua its all numbers so equivalency isn't a problem

Reply:
> > For the enum'd, do you mean like an enum class in luautils and in a lua enum file under scripts/enum?
> 
> Yes. You can still return a uint8 with `static_cast<uint8>(your::enum::val)`, in lua its all numbers so equivalency isn't a problem

Thanks! I updated the PR now to leverage an enum.